Programing-Paradigm-And-Methodology.md

```diff
@@ -0,0 +1,18 @@
+# Programing Paradigm and Methodology
+## Foundational Agile Principles
+**_Individuals and interactions_** over processes and tools
```

Comment:
Reformat using proper markdown using triple asterisks to ***bold and italics***.

```md
***Individuals and interactions*** over ***Processes and tools***
```

> ***Individuals and interactions*** over ***Processes and tools***